### ISSUE TITLE
portable pathnames and seps

### ISSUE THREAD

**Issue by rurban** [perl11/p5-Config#6]
https://github.com/perl11/p5-Config/commit/e6e368c1026ddf6911c8cf696da5f74427d5278c
contains fixes for:

Hardcode the pathsep logic for all supported platforms, w/o File::Spec.

Fix Config_xs.tmp.out: Some platforms cannot handle 2 dots in filenames.

**Comment by rurban** [comment]
see `smoke/6.05_05`

**Comment by bulk88** [comment]
change `Fix Config_xs.tmp.out: Some platforms cannot handle 2 dots in filenames.` I will merge in.

Regarding path separators, did you actually see a failure on some platform? I thought all perls accept `/` or native as input to `open` and friends. Something like `require '../../regen/regen_lib.pl';` or `'../../lib'` could never work if `/` isnt a path sep on PP level.

In perl src in `t/op` dir I see hard coded path seps everywhere

```
BEGIN {
    chdir 't' if -d 't';
    @INC = qw(. ../lib);
    require "./test.pl";
}
```

**Comment by rurban** [comment]
No, but there are still some non-/ pathsep architectures supported.
require fixes that under the hood at run-time.

**Comment by bulk88** [comment]
I dont think the path sep code is really needed, but I shipped it anyways in 6.07 because it didn't hurt anything.
